message ApertureParameters {
  extensions 1000 to max;
}
